### ISSUE TITLE
Drop _pseudo tail for all parse plugins

### ISSUE THREAD

**Issue by radare** [radare/radare2#4317]

**Comment by vient** [comment]
"Drop _pseudo tail for all" but "not all plugins do pseudo"? So all or not all?

**Comment by radare** [comment]
it’s a bug marked as “question” for discussion, so we need to evaluate this and see if we really want to do something else than generating pseudocode with this module

**Comment by wargio** [comment]
i would prefer to keep `_pseudo` and drop `parse_`

**Comment by radare** [comment]
Yeah the thing is that this module is more about pseudocoding instructions than paraing them. If we rename this, pdc logic should be implemented here imho, but this breaks the module dependencies because it will require analysis info.. mmh

**Comment by wargio** [comment]
Oh.. Well that's a problem

**Comment by MaskRay** [comment]
`RParsePlugin::parse` looks strange to me, if I read the code carefully, `parse` is a downstream of `RAsmPlugin::disassemble`, but why not add an extended `disassemble` that emits metadata (structured representation of an instruction) and prettify the metadata in `RParsePlugin::parse`?
